app/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,7 @@
     <string name="do_not_save">Don\'t save</string>
```

Comment:
We should clean up this file when we have time

Reply:
done ;)